src/processors/src/file-handler.ts

```diff
@@ -50,6 +61,44 @@ const JSON_LOADER_OPTIONS = {
   ]
 };
 
+/**
+ * ProcessFileDataContent
+ * @typedef {Object} ProcessFileDataContent
```

Comment:
Nit: We should use typescript types instead of JSDoc types. At minimum remove the types in the comment.

Additionally if you move the docs for the fields into `/** ... */` comment above each field in the type then the vscode will show that on hover.